test/specs/settings_test.js

```diff
@@ -10,4 +11,14 @@ describe('common/settings.js', () => {
     config.teams[0].url.should.equal(v0Config.url);
     config.version.should.equal(settings.version);
   });
+
+  it('should loadDefault config for version 1', () => {
+    const baseConfig = require('../../src/common/config/base.json');
+    const overrideConfig = require('../../src/common/config/override.json');
+    const expectedDefaults = deepmerge(
+      baseConfig[1], overrideConfig[1] || {}, {clone: true, arrayMerge: settings.deepMergeArray}
```

Comment:
This part is actually same to `loadDefault()` implementation. So this test always passes. Is this intended?

Reply:
@yuya-oc I think that it will provide the context to be able to test specific base config and override changes. What other test do you think we should add here?

Reply:
@dmeza I had imagined the test simply comparing to "expected" default config. So in this case, something like `expectedDefaults = { teams: [], showTrayIcon: false, ... }` instead of calling deepmerge() because a plain object is obvious as an expected value.
However, surely this might be enough for now. I can accept.